ws/src/main/java/org/akaza/openclinica/ws/DataEndpoint.java

```diff
@@ -224,6 +224,8 @@ private ODMContainer unmarshallToODMContainer(Element odmElement) throws Excepti
 
         String xml = node2String(odmElement);
         xml = xml.replaceAll("<ODM>", this.ODM_HEADER_NAMESPACE);
+        xml = xml.replaceAll("<!\\[CDATA\\[","");

```

Comment:
CDATA is used to escape special characters in xml. If we remove it completely, this could break the xml parser. I would update the xml parser to handle CDATA. 
Please test with data that has special characters like "<", ">"

Reply:
After did more testing and check the code,

looks like the posted test case xml file and the related OC3 document doesn't match the current OC3 code,

if just simple remove the <!CDATA[* and *]]> tag in the original xml file which was posted in this ticket, should work, in this case no need to do any code change/fix.so just close the PR without merge, we may also need to update the related OC3 document later.